lib/stat_tracker.rb

```diff
@@ -303,4 +362,26 @@ def fewest_goals_scored(team_id)
   def average_win_percentage(team_id)
     win_percentage_per_team[team_id]
   end
+
+  def win_percentage_by_season(team_id)
+    result = {}
+    @game_teams.each do |game_team|
+      result[game_team.season] = win_percentage_per_team
+    end
+    result
+  end
+
```

Comment:
Is win_percentage_by_season a helper method?

Reply:
It is! My bad. I'll move it up to the helper methods section!